frontend/src/App.tsx

```diff
@@ -58,9 +58,9 @@ function App() {
             <Container >
             <Routes>
                 <Route path="/login" element={<Login user={user} login={login}/>}/>
-
+                <Route path="/" element={<Home user={user}/>}/>
                 <Route element={<ProtectedRoute user={user}/>}>
-                    <Route path="/" element={<Home user={user}/>}/>
+                    <Route path="/:id" element={<Home user={user}/>}/>
```

Comment:
Duplicate content: `/`and `:id` both deliver the same page component. You should delete one of them